plugin/trace/trace.go

```diff
@@ -52,7 +52,7 @@ func (t *trace) OnStartup() error {
 		case "zipkin":
 			err = t.setupZipkin()
 		case "datadog":
-			tracer := opentracer.New(tracer.WithAgentAddr(t.Endpoint), tracer.WithServiceName(t.serviceName), tracer.WithDebugMode(true))
+			tracer := opentracer.New(tracer.WithAgentAddr(t.Endpoint), tracer.WithServiceName(t.serviceName), tracer.WithDebugMode(false))
```

Comment:
lgtm, but we can remove the entire `WithDebugmode` option?

Reply:
How about enabling it when debug is on: e.g.  `tracer.WithDebugMode(log.D.Value())`

Reply:
That's even better. Updated PR.